Description: Make sure apport code write files in binary mode.
 Without this patch, dkms_packages.py crashed with AssertionError in
 _assert_bin_mode(): file stream must be in binary mode
Author: Ritesh Raj Sarraf <rrs@debian.org>
Bug-Debian: https://bugs.debian.org/684377
Forwarded: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=685208#10
Reviewed-By: Petter Reinholdtsen <pere@hungry.com>
Last-Update: 2016-05-01

--- dkms-2.2.0.3.orig/dkms_apport.py
+++ dkms-2.2.0.3/dkms_apport.py
@@ -92,4 +92,4 @@ if 'DKMSBuildLog' in report:
 
 if options.kernel:
     report['DKMSKernelVersion'] = options.kernel
-report.write(open(apport.fileutils.make_report_path(report), 'w'))
+report.write(open(apport.fileutils.make_report_path(report), 'wb'))
